### DISCUSSION TITLE
Open new windows via terminal

### DISCUSSION THREAD

**Discussion by onosendi** [wezterm/wezterm#5810] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

Is it possible to spawn new windows via terminal? If I `open -a WezTerm` it just focuses the already opened window. I need this because I want to spawn new terminal windows with a keyboard shortcut.

**Comment by onosendi** [comment, marked as answer]
Looks like `open -na WezTerm` does it.

**Reply by wez** [reply]
See also:
* https://wezfurlong.org/wezterm/cli/start.html
* https://wezfurlong.org/wezterm/cli/cli/spawn.html
* https://wezfurlong.org/wezterm/cli/cli/split-pane.html
* and the other cli subcommands
